cypress/integration/orders/orders.PoSearch.spec.js

```diff
@@ -48,4 +49,16 @@ describe('orders: Test PO search', () => {
         Orders.checkSearchResults(orderNumber);
       });
   });
+  it('C6718 Test the PO filters [except tags]', { tags: [TestType.smoke] }, () => {
+    Orders.createOrderWithOrderLineViaApi(order, orderLine)
```

Comment:
In testrail I see:
```
You'll see a list of various filters possible for Orders:
<li>Status</li>
<li>Approved</li>
<li>Acquisitions unit</li>
<li>Assigned to</li>
<li>Created by</li>
<li>Date created</li>
<li>Date ordered</li>
<li>Order type</li>
<li>Vendor</li>
<li>Tags [ignore this one; covered by another test]</li>
<li>Reason for closure</li>
<li>Re-encumber</li>
<li>Renewal date</li>
<li>Renewal manual</li>
<li>Renewal review period</li>
```

> Test each filter, separately or in combination with others.

I run your test and I can't see checks for:
```
<li>Created by</li>
<li>Date created</li>
<li>Date ordered</li>
<li>Reason for closure</li>
<li>Re-encumber</li>
<li>Renewal date</li>
<li>Renewal manual</li>
<li>Renewal review period</li>
```
May be better to separate test case to test cases for each filter. This test case seems strage about this moment.